src/main/java/com/url/shortener/util/CodeGenerator.java

```diff
@@ -0,0 +1,36 @@
+package com.url.shortener.util;
+
+import com.url.shortener.repository.UrlRepository;
+import org.springframework.stereotype.Component;
+
+import java.util.Random;
+
+@Component
+public class CodeGenerator {
+    private static final String ALPHANUMERIC = "abcdefghijklmnopqrstuvwxyz0123456789";
+    private static final int SHORT_URL_LENGTH = 6;
```

Comment:
ALPHANUMERIC & SHORT_URL_LENGTH are these the defaults or you are gonna set them as env variables later?

Reply:
Hey @a-braham I don't see the need to set these values as env variables because they are not secrets they are just constant values. BUT thanks for the comment🤘🏻

Reply:
Good point.